package.json

```diff
@@ -8,9 +8,11 @@
   },
   "license": "Apache-2.0",
   "scripts": {
-    "dev": "concurrently --kill-others --names \"WEBPACK,SASS\" -c \"blue,magenta\" \"webpack serve --config ./config/webpack/development.config.js\" \"bundle exec rake dartsass:watch\"",
+    "build:css": "sass ./app/assets/stylesheets:./app/assets/builds --no-source-map --load-path=node_modules --load-path=app/assets/stylesheets && sass ./lib/developer_portal/app/assets/stylesheets:./app/assets/builds --no-source-map --load-path=node_modules --load-path=lib/developer_portal/app/assets/stylesheets --load-path=app/assets/stylesheets",
```

Comment:
What is this doing and what is the `sass` executable being called?

Also how does `cssbundling-rails` select the backend sass implementation?

Reply:
I wonder if there is a noticeable performance difference compared to dartsass.

Also do you know how `cssbundling-rails` switches between backend implementations?

Reply:
I don't find the performance differences noticeable, but I also didn't pay close attention. I think as long as it's reasonable (doesn't affect the development process), it's good enough.

I don't understand the question about backend implementations :thinking: